<ui version="4.0" >
 <class>AddMBsSWs_Dialog</class>
 <widget class="QDialog" name="AddMBsSWs_Dialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>489</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font" >
   <font>
    <family>Liberation Sans</family>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Add Measuring Blocks &amp; Switches</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="title_label" >
       <property name="font" >
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Title   [Unit]:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="MBsSWs_listWidget" >
       <property name="minimumSize" >
        <size>
         <width>421</width>
         <height>276</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" >
       <property name="topMargin" >
        <number>0</number>
       </property>
       <item row="0" column="1" >
        <widget class="QPushButton" name="add_pushButton" >
         <property name="minimumSize" >
          <size>
           <width>122</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>122</width>
           <height>32</height>
          </size>
         </property>
         <property name="text" >
          <string>       Add        </string>
         </property>
         <property name="icon" >
          <iconset resource="../resources/FreeSSM.qrc" >
           <normaloff>:/icons/chrystal/32x32/add.png</normaloff>:/icons/chrystal/32x32/add.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="cancel_pushButton" >
         <property name="minimumSize" >
          <size>
           <width>122</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>122</width>
           <height>32</height>
          </size>
         </property>
         <property name="text" >
          <string>    Cancel       </string>
         </property>
         <property name="icon" >
          <iconset resource="../resources/FreeSSM.qrc" >
           <normaloff>:/icons/chrystal/32x32/cancel.png</normaloff>:/icons/chrystal/32x32/cancel.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3" >
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/FreeSSM.qrc" />
 </resources>
 <connections/>
</ui>
